RecoTauTag/HLTProducers/src/L2TauTagFilter.cc

```diff
@@ -55,7 +57,7 @@ class L2TauTagFilter : public HLTFilter {
       throw cms::Exception("Inconsistent Data", "L2TauTagFilter::hltFilter") << "CNN output size != L1 taus size \n";
     }
     for (size_t l1_idx = 0; l1_idx < l1Taus.size(); l1_idx++) {
-      if (L2Outcomes[l1_idx] >= discrWP_) {
+      if (L2Outcomes[l1_idx] >= discrWP_ || l1Taus[l1_idx]->polarP4().pt() > l1PtTh_) {
```

Comment:
Just a question: looks like [`[polarP4().pt()]` is equivalent to `pt()`](https://github.com/cms-sw/cmssw/blob/06f15e3ce91f53b005847b384d4027117598a9d7/DataFormats/Candidate/interface/ParticleState.h#L137), but any reason not to just use `pt()` here?

Reply:
I agree, they seem to be the same. 
@valeriadamante can you comment, please?
@missirol should we change this?

Reply:
I think it's more important to have this PR integrated in `12_2_X` before Tuesday (last pre-release).
Since this change is unconsequential (but please check the values are indeed the same as we expect), I don't see it as a requirement for this PR. That been said, I would do it already here if you can apply this change soon (e.g. today).

Reply:
I have also checked and both `pt()` and `polarP4().pt()` are equval as defined in `reco::LeafCandidate` which base class. There are other occurrences of `polarP4().pt()` in this class - I will try to fix all of them this afternoon.